### PR TITLE
Add login and signup pages

### PR DESCRIPTION
**Pull request by pexomat**

### CODE REVIEW

File: src/components/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            className={style.input}
+          />
+        </div>
+        <button type="submit" className={style.submitButton}>
+          Login
+        </button>
+        <Link href="/">
+          <button className={style.submitButton}>Back</button>
```
Comment (GLObus303) [inline src/components/Login.tsx]: Invalid html

File: src/components/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          <label htmlFor="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            className={style.input}
+          />
+        </div>
+        <button type="submit" className={style.submitButton}>
```
Comment (GLObus303) [inline src/components/Login.tsx]: class naming

File: src/components/Signup.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        <div className={style.inputWrapper}>
+          <label htmlFor="username">First name:</label>
+          <input
+            type="text"
+            id="firstName"
+            value={firstName}
+            onChange={(e) => setFirstName(e.target.value)}
+            className={style.input}
+          />
+        </div>
```
Comment (GLObus303) [inline src/components/Signup.tsx]: Abstract

File: src/pages/event/[id].tsx [hunk 4]
```diff
@@ -31,7 +32,9 @@ const EventDetail: React.FC = () => {
       <p>Name: {event?.name}</p>
       <p>Sport: {event?.sport}</p>
       <p>Id: {id}</p>
-      <button onClick={() => router.push("/")}>Back</button>
+      <Link href="/">
+        <button style={{ cursor: "pointer" }}>Back</button>
```
Comment (GLObus303) [inline src/pages/event/[id].tsx]: pls

**Review by GLObus303 — APPROVED** [review]

**Review by GLObus303 — APPROVED** [review]

File: src/components/Login.tsx (hunk 1 above)
Comment (pexomat) [reply src/components/Login.tsx]: solved

File: src/components/Login.tsx (hunk 2 above)
Comment (pexomat) [reply src/components/Login.tsx]: solved

File: src/components/Signup.tsx (hunk 3 above)
Comment (pexomat) [reply src/components/Signup.tsx]: solved

File: src/pages/event/[id].tsx (hunk 4 above)
Comment (pexomat) [reply src/pages/event/[id].tsx]: solved

File: src/components/Button.module.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
+  text-align: center;
+  font-size: 16px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+
+  &:hover {
+    background-color: #0056b3;
```
Comment (GLObus303) [inline src/components/Button.module.scss]: Add these colors to color.scss, use them from there

**Review by GLObus303 — APPROVED** [review]

**Pull request merged**